packages/plugin/src/theia.d.ts

```diff
@@ -1582,168 +1582,168 @@ declare module '@theia/plugin' {
         waitUntil(thenable: PromiseLike<any>): void;
```

Comment:
I just wanted to mention that this file is never checked by `eslint` due to the following `ignorePatterns`:

https://github.com/eclipse-theia/theia/blob/40b380062f5f9976ba46b38db7c7fe2fa56e9535/configs/base.eslintrc.json#L21-L24

Reply:
maybe this one should not be excluded as it's maintained by us manually

Reply:
> maybe this one should not be excluded as it's maintained by us manually

I think so, we should update the `ignorePatterns` to only check `*d.ts` files located under `lib` folders correct?